src/spyglass/lfp/v1/lfp.py

```diff
@@ -32,15 +32,17 @@ class LFPSelection(dj.Manual):
     The interval list is used to select the times from the raw data that will be
     filtered.  The user can also specify the filter to be used.
 
-    The LFP data is filtered and downsampled to 1 KHz.  The filtered data is stored
-    in the AnalysisNwbfile table.  The valid times for the filtered data are stored
-    in the IntervalList table.
+    The LFP data is filtered and downsampled to the user-defined sampling rate, specified
+    as lfp_sampling_rate.  The filtered data is stored in the AnalysisNwbfile table.
+    The valid times for the filtered data are stored in the IntervalList table.
     """
 
     definition = """
      -> LFPElectrodeGroup                                                  # the group of electrodes to be filtered
      -> IntervalList.proj(target_interval_list_name='interval_list_name')  # the original set of times to be filtered
      -> FirFilterParameters                                                # the filter to be used
+     ---
+     lfp_sampling_rate = 1000 : float                                              # the desired output sampling rate, in HZ                                                                                                                                                                                                                
```

Comment:
@edeno Do you expect this sampling rate to be the same at the one stored in the table below? I'm not sure how the `//` floor operation would work out in all cases. 

```python
decimation = sampling_rate // key["lfp_sampling_rate"]  # this one
key["lfp_sampling_rate"] = sampling_rate // decimation  # next one
```

- If they're the same, I wouldn't want the name overlap to prevent a join of these tables and we can cut down on redundancy by removing from below.
- If either (a) they're different or (b) they're the same, but it's needed as in the part-parent to the Merge table, the I'd rename this one as `target_sampling_rate`

```suggestion
     target_sampling_rate = 1000 : float                                     # the desired output sampling rate, in HZ                                                                                                                                                                                                                
```